shared/src/main/scala/squants/energy/package.scala

```diff
@@ -16,10 +16,10 @@ package squants
 package object energy {
 
   object KineticEnergy {
-    def apply(mass: Mass, velocity: Velocity): Energy =
+    def apply(mass: Mass, velocity: squants.motion.Velocity): Energy =
```

Comment:
Why do we use the full name?

Reply:
We're in the package `squants`, so the only way to avoid picking up the problematic type aliases in the `squants` companion object are fully qualified names.